File: pset6/mario/less/mario.py
# Abstraction and scope

from cs50 import get_int


def main():
    height = get_positive_int("Positive number between 1 and 8 : ")

    max = 1
# Print out that many bricks
    for i in range(height):
        dots = (height-i-1)
        for j in range(dots):
            print(" ", end="")

        for k in range(max):
            print("#", end="")
        print()
        max += 1

def get_positive_int(prompt):
    """Prompt user for positive integer"""
    while True:
        n = get_int(prompt)
        if n > 0 and n < 9 :
            break
    return n




if __name__ == "__main__":
    main()

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int get_positive_int(string prompt);

int main(void)
{
    int h = get_positive_int("Height: ");
    printf("Height : %i \n", h);
    
    int max = 1;

        for (int i = 0; i < h; i++)
        {
            int dots = (h-i);
            for (int k = 1; k < dots; k++)
            {
                    printf(" ");
            }
            
            for (int j = 1; j<= max; j++)
            {
                  printf("#");
            }
          max++;
            
          printf("\n");
        }
}

// Prompt user for a height between 1 and 9
int get_positive_int(string prompt)
{
    int height;
    do{
        height = get_int("%s", prompt);
    }
    while (height < 1 || height > 8);
   // printf("Height value is %d\n", height);
    
    return height;
}

